isapi/install.py

```diff
@@ -228,7 +228,7 @@ def FindWebServer(options, server_desc):
     # make sure server_desc is unicode (could be mbcs if passed in
     #  sys.argv).
     if server_desc and not isinstance(server_desc, str):
-        server_desc = server_desc.decode("mbcs")
+        server_desc = server_desc.decode("utf-8")
```

Comment:
as the comment notes, the source of this is sys.argv - is that now guaranteed to be utf-8? I suspect not.